pkgs/tools/misc/bibutils/default.nix

```diff
@@ -18,13 +18,20 @@ stdenv.mkDerivation rec {
       --replace '-Wl,-soname,$(SONAME)' ""
   '';
 
+  configurePlatforms = [ ];
```

Comment:
What is the default value of this and why is it causing issues?

Reply:
It defaults to `[ "build" "host" ]`, which adds the `--build=<platform>` and `--host=<platform>` flags when cross compiling, however these flags are not recognized by the configure script of bibutils.

Reply:
Not sure if this warrants a comment